mattermost/mattermost-plugin-github
# /api/v1/connected should be publicly accessible

#### Summary
`/api/v1/connected` should be publicly accessible. Otherwise, the webapp can't  figure out if a user is connected or not in the login screen.

#### QA Steps

1. Deploy PR
2. Open Mattermost in an incognito  window
3. Confirm no js error message is shown


#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-github/issues/278
